admin/sql/updates/20160515-recluster.sql

```diff
@@ -0,0 +1,8 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+CLUSTER edit USING edit_pkey;
+
+ANALYZE edit;
```

Comment:
The `CLUSTER` will rewrite the table in order of edit IDs. This is important for the performance of the BRIN indexes from MBS-8804 and will ensure that the data from the dropped columns (most importantly, the huge `data` column) is actually removed from the table. (`ALTER TABLE DROP COLUMN` only hides the column, leaving the actual rows unchanged.)